docs/source/releasenotes.rst

```diff
@@ -43,6 +45,11 @@ Developer Notes
   the scattering matrices or scattered fields needed to be calculated is
   now in `holopy.scattering.imageformation`.
 
+Deprecations
```

Comment:
If you wanted to, you could use the FitWarning in holopy.fitting.py to temporarily maintain the PerfectLensModel namespace but also alert users that it won't stick around.

Reply:
The api is different enough, and the PerfectLensModel was around for short enough, where I'd rather not. 

If you want though, I can do something like

```python
class PerfectLensModel(object):
    def __new__(cls, *args, **kwargs):
        raise NotImplementedError("PerfectLensModel is deprecated; use an AlphaModel with a parameterized lens_angle instead")
```

Or I can leave it as-is (it is only 1 release, and I don't think people used the PerfectLensModel, although maybe I'm wrong.)